src/parse/formats.js

```diff
@@ -1,7 +1,7 @@
 import csvStream from 'csv-parser'
 import excelStream from 'exceljs-transform-stream'
 import through2 from 'through2'
-import shpToJSON from 'shp2json'
```

Comment:
Need to remove this dep now that its unused

Reply:
fixed